Live/Classes/ResourceLayer.cpp

```diff
@@ -23,15 +25,17 @@ ResourceLayer::ResourceLayer() : Layer() {
 
         //This is a hack to avoid the pond on the bottom right of the demo map
         //  as well as the player spawn location
-        while((x >= 1050 && y <= 380) || (std::abs(x-512)<50 && std::abs(y-364) < 50)) {
```

Comment:
Is there anyway to just attempt to check collision using the map manager?

Reply:
might be a good idea, separate PR

Reply:
Please add an issue then.